jre_emul/jre_sources.mk

```diff
@@ -2438,5 +2444,5 @@ JAVA_PUBLIC_SOURCES = $(JAVA_PUBLIC_SOURCES_JRE) $(ANDROID_PUBLIC_SOURCES) \
 JAVA_SOURCES = $(JAVA_PUBLIC_SOURCES) $(JAVA_PRIVATE_SOURCES)
 
 NATIVE_JRE_SOURCES = $(NATIVE_JRE_SOURCES_CORE) $(NATIVE_JRE_SOURCES_ZIP) \
-  $(NATIVE_JRE_SOURCES_CHANNELS) $(NATIVE_JRE_SOURCES_NET) $(NATIVE_JRE_SOURCES_SSL) \
-  $(NATIVE_JRE_ICU_EMBEDDED_DATA)
+  $(NATIVE_JRE_SOURCES_CHANNELS) $(NATIVE_JRE_SOURCES_FILE) $(NATIVE_JRE_SOURCES_NET) $(NATIVE_JRE_SOURCES_SSL) \
+  $(NATIVE_JRE_ICU_EMBEDDED_DATA)
```

Comment:
Please move NATIVE_JRE_SOURCES_SSL to the next line (keeping the backslash on line 2447). We don't strictly enforce line lengths on make files (some paths are just too long), but really long lines are harder to read in some IDEs.

Reply:
Done